### PR TITLE
Placeholder

### PR DESCRIPTION
**Pull request by TrySound**

### CODE REVIEW

**Comment by TrySound** [comment]
Enable travis please.

**Comment by eventualbuddha** [comment]
Enabled Travis and did a lot of this stuff in a different way. If it seems good to you, then we can close this.

**Comment by TrySound** [comment]
@eventualbuddha Why not 0.12? Rollup still support it.

**Comment by TrySound** [comment]
And I have 0.12 on all my machines.

File: test.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 const rollup = require('rollup').rollup;
-const multiEntryPlugin = require('.');
```
Comment (TrySound) [inline test.js]: This doesn't work at least in windows.

File: test.js [hunk 2]
```diff
@@ -82,4 +83,21 @@ describe('rollup-plugin-multi-entry', () => {
   describe('with old API', () => {
     setupTests({ useOldAPI: true });
   });
+
+  it('skipped by filter util', () => {
```
Comment (eventualbuddha) [inline test.js]: I don't understand why this test is needed. We should not be explicitly excluding `entry` in such a test, anyway.

**Comment by TrySound** [comment]
Okay. Don't need, but still add support for 0.12 to be sure everything is okay (if something will be changed).

**Comment by TrySound** [comment]
Publish release please.

**Comment by eventualbuddha** [comment]
Released v1.3.0 with leading NUL in synthetic module id.

**Pull request closed without merging**